streaming/src/main/java/org/apache/spark/streaming/status/api/v1/BatchStatus.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.streaming.status.api.v1;
```

Comment:
actually I think we need to have some discussion on this
the reason I create the package org.apache.spark.streaming.status.api.v1 is because we already have the package o.a.s.streaming.ui
the web interface of streaming seems separate from the spark-core web ui
so i think the api of streaming should do the same.

so it depend on is this api belong to v1
or is this api belong to v1 of streaming?